tasks/cephfs/test_volume_client.py

```diff
@@ -64,6 +64,31 @@ def _configure_vc_auth(self, mount, id_name):
         self._sudo_write_file(mount.client_remote, mount.get_keyring_path(), out)
         self.set_conf("client.{name}".format(name=id_name), "keyring", mount.get_keyring_path())
 
+    def test_default_prefix(self):
+        group_id = "grpid"
+        volume_id = "volid"
+        guest_entity = "guest"
+        DEFAULT_VOL_PREFIX = "/volumes"
+        DEFAULT_NS_PREFIX = "fsvolumens_"
+
+        #create a volume with default prefix
+        self._volume_client_python(self.mount_b, dedent("""
+            vp = VolumePath("{group_id}", "{volume_id}")
+            vc.create_volume(vp, 10, data_isolated=True)
+        """.format(
+            group_id=group_id,
+            volume_id=volume_id,
+            guest_entity=guest_entity
+        )))
+
+        # The dir should be created
+        self.mount_a.stat(os.path.join(DEFAULT_VOL_PREFIX, group_id, volume_id))
```

Comment:
It's currently incorrectly stat-ing "/volumes/grpid/volid" and not "volumes/grpid/volid" from within mount_a's mountpoint due to incorrect value of DEFAULT_VOL_PREFIX.